Packs/FortinetFortiwebVM/Integrations/FortinetFortiwebVM/FortinetFortiwebVM.yml

```diff
@@ -0,0 +1,2407 @@
+      isArray: false
+      name: block_period
+      required: false
+    - auto: PREDEFINED
+      description: "Select which severity level the FortiWeb appliance will use when a blacklisted IP address attempts to connect to your web servers. Supports API version 2 only."
+      isArray: false
+      name: severity
+      predefined:
+      - Low
+      - Medium
+      - High
+      - Info
+      required: false
+    - auto: PREDEFINED
+      description: "enable Ignore X-Forwarded-For so that the IP addresses will be scanned at the TCP layer instead of HTTP layer. Supports API version 2 only."
+      isArray: false
+      name: ignore_x_forwarded_for
+      predefined:
+      - enable
+      - disable
+      required: false
+    - description: "Select which trigger, if any, that the FortiWeb appliance will use when it logs and/or sends an alert email about a blacklisted IP address’s attempt to connect to your web serversSupports API version 2 only."
+      name: trigger_policy
+      required: false
+    deprecated: false
+    description: Update IP List.
+    execution: false
+    name: fortiwebvm-ip-list-group-update
+  - arguments:
+    - description: IP List Name.
+      name: name
+    - description: The page number of the results to retrieve.
+      isArray: false
+      name: page
+      required: false
+    - description: The maximum number of records to retrieve per page.
+      isArray: false
+      name: page_size
+      required: false
+    - defaultValue: '50'
+      description: The maximum number of records to retrieve.
+      isArray: false
+      name: limit
+      required: false
+    deprecated: false
+    description: Supports API versions 1 & 2.
+    execution: false
+    name: fortiwebvm-ip-list-group-list
+    outputs:
+    - contextPath: FortiwebVM.IpListGroup.id
+      description: IP list id.
+      type: String
+    - contextPath: FortiwebVM.IpListGroup.ip_list_count
+      description: IP list members number.
+      type: Number
+    - contextPath: FortiwebVM.IpListGroup.can_delete
+      description: Can delete flag. Supports API version 1 only.
+      type: Boolean
+    - contextPath: FortiwebVM.IpListGroup.q_ref
+      description: The CMDB reference count. Supports API version 2 only.
+      type: Number
+    - contextPath: FortiwebVM.IpListGroup.q_type
+      description: IP list group object type. Supports API version 2 only.
+      type: Number
+    - contextPath: FortiwebVM.IpListGroup.can_clone
+      description: Can clone flag. Supports API version 2 only.
+      type: Number
+    - contextPath: FortiwebVM.IpListGroup.block_period
+      description: IP list group Block Period. Supports API version 2 only.
+      type: Number
+    - contextPath: FortiwebVM.IpListGroup.can_view
+      description: Can view flag. Supports API version 2 only.
```

Comment:
Don't understand this description. The type is a number.  What is the correct description? Should it be: "Whether you can view the IP list group." ? Is the value of this 0 or 1?

Reply:
Yes,  and the value is 0 or 1.